src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends FigureBase {
+    private int side;
+
+    public Square(String color, int side) {
```

Comment:
Don't pass expressions in the constructor. Create separate variables and pass them on for better code readability. For example, assign 'color' and 'side' to variables before passing them to the constructor.